Lib/idlelib/browser.py

```diff
@@ -19,14 +19,45 @@
 from idlelib.tree import TreeNode, TreeItem, ScrolledCanvas
 from idlelib.windows import ListedToplevel
 
+
 file_open = None  # Method...Item and Class...Item use this.
 # Normally pyshell.flist.open, but there is no pyshell.flist for htest.
 
+
+def _traverse_node(node, name=None):
+    """Return the immediate children for a node.
+
+    Node is the current node being traversed.  The return value is
+    a tuple with the first value being a dictionary of the
+    Class/Function instances of the node and the second is
+    a list of tuples with (lineno, name).
+    """
+    items = []
+    children = {}
+    for key, cl in node.items():
+        if name is None or cl.module == name:
+            s = key
+            if hasattr(cl, 'super') and cl.super:
+                supers = []
+                for sup in cl.super:
+                    if type(sup) is type(''):
+                        sname = sup
+                    else:
+                        sname = sup.name
+                        if sup.module != cl.module:
+                            sname = f'{sup.module}.{sname}'
+                    supers.append(sname)
+                s += '({})'.format(', '.join(supers))
+            items.append((cl.lineno, s))
+            children[s] = cl
+    return children, items
+
+
 class ClassBrowser:
```

Comment:
This should become ModuleBrowser here, 3 other places in file, and in pathbrowser.  'Class Browser' should also be change in mainmenu.  We should do this in a separate issue and patch.  Lets see if we can do this quickly.

Reply:
Did you want the change to ModuleBrowser to happen before this current PR?